profile/README.md

```diff
@@ -1,17 +1,28 @@
 # Project-Origin
+<!--
+The mission of the project in as a concise format as possible. It should answer what the project's purpose is and enable contributors to make a good first guess whether a suggested feature will likely be in scope for the project, or not.
+-->
 
-Project-Origin aims to create solutions to enable publicly verifiable proofs for tracking of Energy.
+Project-Origin is an open source collaboration that aims to create a public permissioned distributed ledger where everyone can validate the origin of their electricity. By being able to do so, we can support and accelerate the green transition. The long-term goal is to expand the scope from electricity, to include other utilities, because we find that the functionality can create value in a broader scope. By creating an open source platform for collaboration, we can pool ressources and accelerate the development of critical digital infrastructure that is secure and the data verifiable. 
 
-## Registry
+The open source platform is set up with decision- and work structures, to achieve effective asynchronous collaboration through transparent, accessible software, and by using decision-making processes driven by \emph{doing}.
 
-The goal of [this project](https://github.com/orgs/project-origin/projects/1) is to create the first version of a **Federated Publicly Verifiable Registry**
+## Ongoing Implementation Efforts 
+There is currently an implementation effort working on creating the first version of a federated, publicly verifiable registry. Go to the [Registry repository](https://github.com/project-origin/registry) to learn more about the implementation details and project status. 
 
-The registry will enable the following:
+## Getting Involved
```

Comment:
I think this is a good section. I would also point towards the individual README and CONTRIBUTING files for further information on specific projects. 
Discussion channels are good, I wonder if we should not also highlight the readymade project boards?
As for the "how to start collaborating with us" question - some onboarding docs, more general documentation, a filter with "good first issues", regular meeting links, and places where we're looking for help or input would be great. Maybe this could be something to place in the [documentation](https://github.com/project-origin/documentation) repository?

Reply:
Good inputs here - I will work on adding these changes

Reply:
I have put a link for the documentation, even though onboarding docs are not yet there. I will create some issues and put them in the project board :-)